subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -38,18 +32,6 @@
                 "Method has been removed"
             ]
         },
-        {
-            "type": "org.gradle.api.publish.maven.tasks.AbstractPublishToMaven",
-            "member": "Method org.gradle.api.publish.maven.tasks.AbstractPublishToMaven.getDuplicatePublicationTracker()",
-            "acceptation": "Injected service",
-            "changes": []
-        },
-        {
-            "type": "org.gradle.api.publish.ivy.tasks.PublishToIvyRepository",
-            "member": "Method org.gradle.api.publish.ivy.tasks.PublishToIvyRepository.getDuplicatePublicationTracker()",
-            "acceptation": "Injected service",
-            "changes": []
-        },
         {
```

Comment:
Can we remove the `GroovyCompile` entry as well: https://github.com/gradle/gradle/blob/master/subprojects/distributions/src/changes/accepted-public-api-changes.json#L83-L88

Reply:
Done.